husky_components/datagrid/decorators/table-view.js

```diff
@@ -1424,7 +1424,7 @@ define(function() {
          * @param event {Object} the event object
          */
         allSelectItemChangeHandler: function(event) {
-            if (this.data.total == 0) {
+            if (this.data.embedded.length == 0) {
```

Comment:
I guess this only changed because you are not returning a `ListRepresentation` as you've written in https://github.com/sulu/sulu/pull/2704#discussion_r72930210, right? Maybe we should return it there, and leave this the same, so that it also works with pagination?